src/app/plugins-initializer/plugins-initializer.ts

```diff
@@ -0,0 +1,64 @@
+import { APP_INITIALIZER, Compiler, FactoryProvider, Injector } from '@angular/core';
+import { A1PluginDefinition } from './shared/a1-plugin-definition';
```

Comment:
I missed that we comitted this to step-frontend, lets rename the plugin classes to something meaningful